src/transport/webrtc/mod.rs

```diff
@@ -712,3 +728,9 @@ impl Stream for WebRtcTransport {
             .map_or(Poll::Pending, |event| Poll::Ready(Some(event)))
     }
 }
+
+// Check if the packet received is STUN.
```

Comment:
Is there a spec to reference to explain the check below?

Reply:
Yep, I ve added a link to the docs: https://datatracker.ietf.org/doc/html/rfc5389#page-10